pixl_core/src/core/project_config.py

```diff
@@ -60,6 +60,28 @@ class _Project(BaseModel):
     modalities: list[str]
 
 
+class TagOperations(BaseModel):
+    """Tag operations files for a project. At least a base file is required."""
+
+    base: Path
```

Comment:
I think this should probably be a list if we want to keep the same API for allowing merging of schemes open for the future